src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
Sorry, I don't understand. Did not match with what? You mean the the views were some exotic object, not derived from `Backbone.View`?